provisioning/baremetal_pod.go

```diff
@@ -263,18 +263,30 @@ func createInitContainerIpaDownloader(images *Images) corev1.Container {
 	return initContainer
 }
 
-func createInitContainerMachineOsDownloader(images *Images, config *metal3iov1alpha1.ProvisioningSpec) corev1.Container {
+func createInitContainerMachineOsDownloader(info *ProvisioningInfo) corev1.Container {
+	optionValue := ""
+	switch info.NetworkStack {
```

Comment:
Isn't NetworkStack a bitfield due to https://github.com/openshift/cluster-baremetal-operator/pull/148/files#diff-7ad72d08a06fe28b03a15c84289b815f693443562d0c67e6f461c7e3f5ee0707R315? This isn't going to give the right result in dual-stack scenarios since neither option will match.

Reply:
Right, we don't hit either case here, so it's left blank which is the behavior for dualstack.  The behavior here is more implicit, I think it'd be clearer if there was a case for NetworkStackDualstack.

Reply:
https://github.com/openshift/cluster-baremetal-operator/pull/154